### ISSUE TITLE
Bug: GraphiQL headers not set when new query pane 

### ISSUE THREAD

**Issue by dthyresson** [redwoodjs/studio#26]
Headers for user impersonation are set in the initial pane. But it you + to make anew pane and query, the headers are not set. A page refresh sets them.

May need to ad a custom button to set headers or intercept the add new query and set.

**Comment by Tobbe** [comment]
Yes, this is something I've noticed too. But I didn't look into it more because we're on a really old version of the Playground. If/when we build any custom behavior it should be on top of the latest version of the Playground.
